quicktests/overlaying/tests/basic/grid_plot.js

```diff
@@ -1,32 +0,0 @@
-
-function makeData() {
-  "use strict";
-
-    var data = [{name: "jon", y: 10, type: "q1"}, {name: "dan", y: 2, type: "q1"}, {name: "zoo", y: 1, type: "q1"}];
```

Comment:
This part caught #2014, but I think we should accomplish that by having messy/wrong-type data in other tests.